website/docs/language/values/variables.html.md

```diff
@@ -303,6 +304,23 @@ random_pet.animal: Creating...
 random_pet.animal: Creation complete after 0s [id=jae-known-mongoose]
 ```
 
+### Disallowing null Module Input Values
+
+[inpage-nullable]: #disallowing-null-input-values
+
+-> This feature was introduced in Terraform v1.1.0.
+
+The default value for `nullable` is `true`. When `nullable` is `true`, `null`
```

Comment:
I think it'd be good to have an introductory paragraph here that allows the reader to know we're about to talk about an argument called `nullable`, before getting into the details:

> The `nullable` argument in a `variable` block controls whether the module caller may assign the value `null` to the variable:
>
> ```hcl
> variable "example" {
>   type     = string
>   nullable = false 
> }
> ```

...and then from there keep these existing paragraphs, starting with "The default value for `nullable` is ...", so that hopefully by the time we're reading that we understand what exactly we mean by `nullable` in this sentence.

Reply:
For what it's worth, I agree with @apparentlymart here! That's a great suggestion!